camunda-formio-tasklist-vue/src/components/layout/LeftSider.vue

```diff
@@ -131,6 +134,12 @@ export default class LeftSider extends Mixins(BaseMixin) {
   @Prop() private selectedfilterId!: string;
   @Prop() private payload!: Payload;
   @Prop() private taskLoading!: boolean;
+  @Prop ({
+    type: Object ,
+    default :{
+      filterList :true ,filterTask:true ,form:true,sort:true
+    }
+  }) private disableComponents!: SortPayload ;
```

Comment:
SortPayload -> rename to something more meaning.

Maybe: disableComponentPropPayload